backend/listener.js

```diff
@@ -129,22 +129,32 @@ async function connectWS({ network }) {
     } else if (data.id === 3) {
       log.info(`Got ${data.result.length} unhandled logs`)
       for (const result of data.result) {
-        await handleLog({
-          ...result,
-          web3,
+        await eventsQueue.add(
+          {
+            ...result,
+            address,
+            networkId,
+            contractVersion
+          },
+          {
+            attempts: 3,
+            jobId: result.transactionHash
```

Comment:
nit: it's not obvious for folks not deeply familiar with bull so would be useful to have a comment explaining that the bull queue is idempotent and that as multiple instances of the listener enqueue the same jobId, only one gets executed.

Reply:
Sure, added a comment.